Add Support for SentryJavascript 7.15.0

https://github.com/getsentry/sentry-javascript/blob/master/CHANGELOG.md#7150

This version introduced some break changes to the code so a new release will required in order to support it. 

```

sentry-capacitor\src\scope.ts
  9:37  error  'Scope' is deprecated. This export has moved to  deprecation/deprecation

sentry-capacitor\src\sdk.ts
  40:28  error  'Hub' is deprecated. This export has moved to       deprecation/deprecation
  41:3   error  'makeMain' is deprecated. This export has moved to  deprecation/deprecation
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,6 +10,9 @@
 - Bump Sentry Android SDK to `6.4.3` ([#230](https://github.com/getsentry/sentry-capacitor/pull/230))
   - [changelog](https://github.com/getsentry/sentry-java/releases/tag/6.4.3)
   - [diff](https://github.com/getsentry/sentry-java/compare/5.7.0...6.4.3)
+- Bump Sentry JavaScript SDK to `7.15.0` ([#???](https://github.com/getsentry/sentry-capacitor/pull/???))
+  - [changelog](https://github.com/getsentry/sentry-javascript/releases/tag/7.15.0)
+  - [diff](https://github.com/getsentry/sentry-javascript/compare/7.13.0...7.15.0)
 
 ## 0.10.1
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,15 +4,15 @@
 
 ### Dependencies
 
+- Bump Sentry JavaScript SDK to `7.15.0` ([#244](https://github.com/getsentry/sentry-capacitor/pull/244))
+  - [changelog](https://github.com/getsentry/sentry-javascript/releases/tag/7.15.0)
+  - [diff](https://github.com/getsentry/sentry-javascript/compare/7.13.0...7.15.0)
 - Bump Sentry iOS SDK to `7.27.1` ([#231](https://github.com/getsentry/sentry-capacitor/pull/231))
   - [changelog](https://github.com/getsentry/sentry-cocoa/releases/tag/7.27.1)
   - [diff](https://github.com/getsentry/sentry-cocoa/compare/7.23.0...7.27.1)
 - Bump Sentry Android SDK to `6.4.3` ([#230](https://github.com/getsentry/sentry-capacitor/pull/230))
   - [changelog](https://github.com/getsentry/sentry-java/releases/tag/6.4.3)
   - [diff](https://github.com/getsentry/sentry-java/compare/5.7.0...6.4.3)
-- Bump Sentry JavaScript SDK to `7.15.0` ([#???](https://github.com/getsentry/sentry-capacitor/pull/???))
-  - [changelog](https://github.com/getsentry/sentry-javascript/releases/tag/7.15.0)
-  - [diff](https://github.com/getsentry/sentry-javascript/compare/7.13.0...7.15.0)
 
 ## 0.10.1
 

--- a/example/ionic-angular/package.json
+++ b/example/ionic-angular/package.json
@@ -14,7 +14,7 @@
     "@ionic-native/splash-screen": "^5.0.0",
     "@ionic-native/status-bar": "^5.0.0",
     "@ionic/angular": "^5.0.0",
-    "@sentry/angular": "^7.13.0",
+    "@sentry/angular": "7.15.0",
     "@sentry/capacitor": "file:.yalc/@sentry/capacitor",
     "rxjs": "~6.5.5",
     "tslib": "^2.0.0",

--- a/package.json
+++ b/package.json
@@ -37,9 +37,9 @@
   "license": "MIT",
   "peerDependencies": {
     "@capacitor/core": ">=3.0.0",
-    "@sentry/angular": "7.13.0",
-    "@sentry/react": "7.13.0",
-    "@sentry/vue": "7.13.0"
+    "@sentry/angular": "7.15.0",
+    "@sentry/react": "7.15.0",
+    "@sentry/vue": "7.15.0"
   },
   "peerDependenciesMeta": {
     "@sentry/angular": {
@@ -53,13 +53,13 @@
     }
   },
   "dependencies": {
-    "@sentry/browser": "7.13.0",
-    "@sentry/core": "7.13.0",
-    "@sentry/hub": "7.13.0",
-    "@sentry/integrations": "7.13.0",
-    "@sentry/tracing": "7.13.0",
-    "@sentry/types": "7.13.0",
-    "@sentry/utils": "7.13.0",
+    "@sentry/browser": "7.15.0",
+    "@sentry/core": "7.15.0",
+    "@sentry/hub": "7.15.0",
+    "@sentry/integrations": "7.15.0",
+    "@sentry/tracing": "7.15.0",
+    "@sentry/types": "7.15.0",
+    "@sentry/utils": "7.15.0",
     "@sentry/wizard": "^1.1.4",
     "promise": "^8.1.0"
   },
@@ -68,8 +68,8 @@
     "@capacitor/core": "^3.0.1 || ^4.0.0",
     "@capacitor/ios": "^3.0.1 || ^4.0.0",
     "@ionic/prettier-config": "^1.0.0",
-    "@sentry-internal/eslint-config-sdk": "7.13.0",
-    "@sentry-internal/eslint-plugin-sdk": "7.13.0",
+    "@sentry-internal/eslint-config-sdk": "7.15.0",
+    "@sentry-internal/eslint-plugin-sdk": "7.15.0",
     "@sentry/typescript": "5.20.1",
     "@types/jest": "^26.0.15",
     "eslint": "^7.13.0",

--- a/src/scope.ts
+++ b/src/scope.ts
@@ -1,4 +1,4 @@
-import { Scope } from '@sentry/hub';
+import { Scope } from '@sentry/core';
 import { Breadcrumb, User } from '@sentry/types';
 
 import { NATIVE } from './wrapper';

--- a/src/sdk.ts
+++ b/src/sdk.ts
@@ -3,7 +3,7 @@ import {
   init as browserInit,
   StackFrame,
 } from '@sentry/browser';
-import { Hub, makeMain } from '@sentry/hub';
+import { Hub, makeMain } from '@sentry/core';
 import { RewriteFrames } from '@sentry/integrations';
 
 import { DeviceContext, EventOrigin, SdkInfo } from './integrations';

